### PR TITLE
Add github issue title fetcher

### LINKED ISSUE #7: Add github issue title fetcher
## Description
After asking the user the issue that they are working on, we should display the Github Issue title using the Github API so that the users can validate the issue number.

### PR DESCRIPTION
**Pull request by dotrachit**
## Issue Fix
Fixes #7 

## Description
* Added two questions:-
1. Which issue are you working on today? (String)
2. What is the type of issue? (MCQ)
* Fetches GH Issue title based on the issue number entered.

## GIF
Screen Recording 2020-11-18 at 5 49 04 PM mov

### CODE REVIEW

**Review by agg-shambhavi — APPROVED** [review]
LGTM 🚀

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,2 @@
 .vscode/
-node_modules/
+node_modules/
```
Comment (yashk2000) [inline .gitignore:2]: Any reason to remove the extra line?

File: lib/files.js [hunk 2]
```diff
@@ -3,10 +3,12 @@ const path = require('path');
...
     getCurrentDirectoryBase: () => {
+        // gets the basename of the directory
         return path.basename(process.cwd());
     },
 
     directoryExists: (filePath) => {
+        // gets the whole directory list
         return fs.existsSync(filePath);
     }
 };
```
Comment (yashk2000) [inline lib/files.js:14]: Let's put extra lines at end of every file?

File: .gitignore (hunk 1 above)
Comment (dotrachit) [reply .gitignore:2]: my bad

File: lib/files.js (hunk 2 above)
Comment (dotrachit) [reply lib/files.js:14]: yup!

**Review by yashk2000 — APPROVED** [review]

**Pull request merged**